Dashboard/AirSectionPickerView.swift

```diff
@@ -0,0 +1,55 @@
+// Created by Lunar on 06/05/2021.
+//
+
+import SwiftUI
+
+struct AirSectionPickerView: View {
+    
+    @Binding var selection: SelectedSection
+    
+    var body: some View {
+        ScrollView(.horizontal, showsIndicators: false) {
+            HStack {
+                ForEach(SelectedSection.allCases, id: \.self) { section in
+                    Button(section.rawValue) {
+                        withAnimation(.easeInOut(duration: 0.1)) {
+                            selection = section
+                        }
+                    }
+                    .buttonStyle(PickerButtonStyle(isSelected: section == selection))
+                }
+            }
+        }
+    }
+}
+
+struct AirSectionPickerView_Previews: PreviewProvider {
+    static var previews: some View {
+        AirSectionPickerView(selection: .constant(.mobileActive))
+            .padding()
+            .frame(width: 300)
+            .previewLayout(.sizeThatFits)
+    }
+}
+
+enum SelectedSection: String, CaseIterable {
+    case following = "Following"
+    case mobileActive = "Mobile active"
+    case mobileDormant = "Mobile dormant"
+    case fixed = "Fixed"
```

Comment:
are the raw string values to be displayed to the user? Maybe a good idea to use LocalilzedString(